e2e/scripts/validate-generated-project.js

```
@@ -72,13 +72,13 @@ const main = async (opts) => {
     }
 
     try {
-        console.log(await validateGeneratedArtifacts(project))
-        if (project === 'retail-app-ext' || project === 'retail-app-ext') {
-            console.log(await validateExtensibilityConfig(project, templateVersion))
+        await validateGeneratedArtifacts(project)
+        if (project === 'retail-app-ext') {
+            await validateExtensibilityConfig(project, templateVersion)
         }
     } catch (err) {
         console.error(err)
-        process.exit(1)
+        // process.exit(1)
```

Just double checking: do we really not want to exit with the error status?

---

Yeah I replaced it with a throw err.

I don't know why both those function calls were wrapped in a console.log()
That console log was causing the exceptions to be swallowed.
I removed that. So now without explicit process.exit, if the catch block is hit, it should propagate upwards.